care/emr/tasks/cleanup_incomplete_file_uploads.py

```diff
@@ -0,0 +1,48 @@
+from datetime import timedelta
+from logging import Logger
+
+from botocore.exceptions import ClientError
+from celery import shared_task
+from celery.utils.log import get_task_logger
+from django.conf import settings
+from django.utils import timezone
+
+from care.emr.models import FileUpload
+
+logger: Logger = get_task_logger(__name__)
+
+
+@shared_task()
+def cleanup_incomplete_file_uploads():
+    """
+    Hard-deletes FileUpload objects that have not been completed.
+    """
+    threshold = timezone.now() - timedelta(hours=settings.FILE_UPLOAD_EXPIRY_HOURS)
+    logger.info("Cleaning up incomplete file uploads")
+    page_size = 1000
```

Comment:
No need for pagination, let's do this one by one.

Reply:
S3 supports batching 
So it would be better to do that to reduce the number of api calls

Reply:
thanks to gcp we cant have nice things